mlir/test/Dialect/SCF/transform-ops.mlir

```diff
@@ -270,3 +270,32 @@ module attributes {transform.with_named_sequence} {
     transform.yield
   }
 }
+
+// -----
+
+// CHECK-LABEL: func @coalesce_i32_loops(
+
+func.func @coalesce_i32_loops() {
+  %0 = arith.constant 0 : i32
+  %1 = arith.constant 128 : i32
+  %2 = arith.constant 2 : i32
+  %3 = arith.constant 64 : i32
+  // CHECK: %[[C0_I32:.*]] = arith.constant 0 : i32
```

Comment:
I think, constants order are not guaranteed either, so better to use `// CHECK-DAG:`

Reply:
Done